src/geo/lng_lat.js

```diff
@@ -121,6 +127,13 @@ class LngLat {
             new LngLat(this.lng + lngAccuracy, this.lat + latAccuracy));
     }
 
+    toEcef(altitude: number): [number, number, number] {
+        const altInEcef = globeMetersToEcef(altitude);
+        const radius = GLOBE_RADIUS + altInEcef;
+        const ecef = latLngToECEF(this.lat, this.lng, radius);
+        return [ecef[0], ecef[1], ecef[2]];
```

Comment:
Nit: `latLngToECEF` already returns a new 3-item array, there seems to be no purpose in making another one.